requirements.txt

```diff
@@ -1,3 +1,4 @@
 anymarkup
 lockfile
 jsonpointer
+requests
```

Comment:
we still have to do it in `requirements.txt` since our Debian Dockerfile still installs via `requirements.txt` :)